packages/reactstrap-validation-select/resources.js

```diff
@@ -46,6 +47,46 @@ const AvCodeSelect = AvResourceSelect.create({
   requiredParams: ['list'],
   watchParams: ['list'],
 });
+const AvPatientSelect = AvResourceSelect.create({
+  resource: avThanosApi,
+  getOptionLabel: option =>
+    `${option.lastName}, ${option.firstName} (${option.memberId ||
+      option.subscriberMemberId})`,
+  requiredParams: ['customerId'],
+  watchParams: ['customerId', 'providerUserId'],
+  hasMore: false, // pagination not supported
+  getResult: data => data.data.patientsMany,
+  graphqlConfig: {
+    type: 'patient',
+    query: `query($filters: PatientFilters) {
+  patientsMany(filters: $filters) {
```

Comment:
Shouldn't we be using the `paginationQuery` here?

Reply:
The `coverages` api, which is what `thanos` is pulling patient data from (for now) - does not support pagination. So the `patients` query in `thanos` doesn't conform to the spec of other aries 1 api queries.

Reply:
Ah sounds good, I am going to remove `thanos` from `sdk-js` and rename it differently as its going to replace all of those endpoints eventually. The name is going to change as well since copyright.